test/service/test_repository_proxy.py

```diff
@@ -248,13 +245,10 @@ def test_repository_files_proxy(self, dss_direct_access_role):
                                                               status=301,
                                                               headers={'Location': dss_url_with_token.url,
```

Comment:
```suggestion
                                                              headers={'Location': str(dss_url_with_token),
```

Reply:
@hannes-ucsc it seems like there are some `url.url` instances of `furl` in `integration_test.py`, should we also address them in this PR? I am thinking yes, but just want to confirm.